meinberlin/assets/js/swiper_phases.js

```diff
@@ -1,5 +1,12 @@
-import Swiper from 'swiper/bundle'
-import 'swiper/css/bundle'
+import Swiper from 'swiper'
+
+// Needed due to issue with Swiper, issue link: https://github.com/import-js/eslint-plugin-import/issues/2266
+/* eslint-disable import/no-unresolved */
+import { Navigation, Pagination } from 'swiper/modules'
+import 'swiper/css'
+import 'swiper/css/navigation'
+import 'swiper/css/pagination'
+/* eslint-enable import/no-unresolved */
```

Comment:
This is apparently a known issue, have added link, also just did a little update to only import the used modules as the bundle is a bit big